### PR TITLE
Need to merge stringsandlists

### PR DESCRIPTION
**Pull request by srmoore5**
http://srmoore5.github.io/spring2014/2014/01/31/stringandlistsierra.html

It is live!

### CODE REVIEW

**Comment by eah13** [comment]
@srmoore5 these exercises don't have outpust listed in code blocks

**Comment by srmoore5** [comment]
It has been corrected now.

**Comment by gerbal** [comment]
@srmoore5 your last code block broke because it doesn't have a newline before the ```

File: _posts/2014-01-31-stringsandlistssierra.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+testEqual(replace(s, 'om', 'am'),
+       'I love spam!  Spam is my favorite food.  Spam, spam, spam, yum!')
+
+testEqual(replace(s, 'o', 'a'),
+
+       'I love spam!  Spam is my favarite faad.  Spam, spam, spam, yum!')
+       
+```
+result:
+```
```
Comment (eah13) [inline _posts/2014-01-31-stringsandlistssierra.md:197]: @srmoore5 add a blank line (aka newline) at this point in the file then you're good to go.

**Comment by srmoore5** [comment]
Fixes #213

**Comment by eah13** [comment]
Looks good.

**Pull request merged**